src/screens/WebView/HelpCenter.js

```diff
@@ -4,6 +4,7 @@ import {WebView} from 'react-native-webview';
 
 import Header from '../../components/Header';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from the `utils/theme` module is a good change. It helps to maintain consistency in color usage across the application and makes it easier to manage color changes in the future.
```diff
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->